frontend/src/PageNotFound.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom';
+import { DeleteButton } from './components/DeleteButton';
+import { PoweredBy } from './components/PoweredBy';
+import { TwitterButton } from './components/TwitterButton';
+
+export const PageNotFound = () => {
+    return (
+            <div className="w-full flex flex-col items-center h-screen text-center">
+                {/* <div className="grid grid-cols-12 gap-4"> */}
+
+                <div className="col-span-12 text-4xl sm:text-6xl mt-auto text-primary">
+                    <span className="text-8xl">404</span>
+                    <br />Page Not Found
+                </div>
+
+                <div className="text-white text-lg col-span-12 mt-2">
+                    The page you are looking for does not exist.
+                </div>
+
+                <Link to={'/'} className="text-primary brightness-75 text-lg mt-2 hover:font-bold">Go Home</Link>
```

Comment:
Remove unnecessary {}, instead have `to='/'`

Reply:
Other links have a white underline on hover. This one has `font-bold`. Keep them the same and make this white underline on hover.